src/index.js

```diff
@@ -382,6 +382,7 @@ exports.collection = (name, opts) => {
   }
 };
 
+exports.SchemaSerializer = require('./serializers/schema');
```

Comment:
It's already required earlier. 
```suggestion
exports.SchemaSerializer = SchemaSerializer;
```